deployments/examples/ocis_full/.env

```
@@ -198,7 +198,7 @@ WOPISERVER_ONLYOFFICE_DOMAIN=
 # Inbucket is a mail catcher tool for testing purposes.
 # DO NOT use in Production.
 # Note: the leading colon is required to enable the service.
-#INBUCKET=:inbucket.yml
+# INBUCKET=:inbucket.yml
```

```suggestion
#INBUCKET=:inbucket.yml
```
The blank not present is intentionally, pls leave as it is.